assets/xml/objects/object_bigpo.xml

```diff
@@ -1,47 +1,47 @@
-        <Texture Name="gBigpoArmSectionTex" OutName="bigpo_arm_section" Format="rgba16" Width="16" Height="16" Offset="0x2870" />
-        <Texture Name="gBigpoHoodEdge" OutName="bigpo_hood_edge" Format="rgba16" Width="8" Height="8" Offset="0x2A70" />
-        <Texture Name="gBigpoNecklaceBeadsTex" OutName="bigpo_necklace_beads" Format="rgba16" Width="16" Height="8" Offset="0x2AF0" />
-        <Texture Name="gBigpoShawlTex" OutName="bigpo_shawl" Format="rgba16" Width="16" Height="16" Offset="0x2BF0" />
-        <Texture Name="gBigpoGlowingEyeTex" OutName="bigpo_glowing_eye" Format="rgba16" Width="16" Height="16" Offset="0x2DF0" />
+        <Animation Name="gBigPoeSwingLampAttackAnim" Offset="0x158" /> <!-- Original name is "pog_atack" -->
+        <Animation Name="gBigPoeShockAnim" Offset="0x454" /> <!-- Original name is "pog_damage" -->
+        <Animation Name="gBigPoeBackpeddleAnim" Offset="0x608" /> <!-- Original name is "pog_escape" -->
+        <Animation Name="gBigPoeFloatAnim" Offset="0x924" /> <!-- Original name is "pog_fly" -->
+        <Animation Name="gBigPoeAwakenSpawnAnim" Offset="0xF9C" /> <!-- Original name is "pog_start" -->
+        <Animation Name="gBigPoeAwakenStretchAnim" Offset="0x1360" /> <!-- Original name is "pog_wait" -->
+        <Texture Name="gBigPoeSoulFace" OutName="bigpo_soul_face" Format="i8" Width="32" Height="64" Offset="0x1370" />
+        <DList Name="gBigPoeDrawSoulDL" Offset="0x1BB0" /> <!-- Original name is "bigpo_soul_modelT" -->
+        <Texture Name="gBigPoeNecklaceSkullHalfTex" OutName="bigpo_skull_necklace" Format="rgba16" Width="16" Height="16" Offset="0x1C70" />
+        <Texture Name="gBigPoeLanternGlassTex" OutName="bigpo_lantern_glass" Format="rgba16" Width="16" Height="16" Offset="0x1E70" />
+        <Texture Name="gBigPoeTornRobeTex" OutName="bigpo_torn_robe" Format="rgba16" Width="32" Height="32" Offset="0x2070" />
+        <Texture Name="gBigPoeArmSectionTex" OutName="bigpo_arm_section" Format="rgba16" Width="16" Height="16" Offset="0x2870" />
+        <Texture Name="gBigPoeHoodEdge" OutName="bigpo_hood_edge" Format="rgba16" Width="8" Height="8" Offset="0x2A70" />
+        <Texture Name="gBigPoeNecklaceBeadsTex" OutName="bigpo_necklace_beads" Format="rgba16" Width="16" Height="8" Offset="0x2AF0" />
+        <Texture Name="gBigPoeShawlTex" OutName="bigpo_shawl" Format="rgba16" Width="16" Height="16" Offset="0x2BF0" />
+        <Texture Name="gBigPoeGlowingEyeTex" OutName="bigpo_glowing_eye" Format="rgba16" Width="16" Height="16" Offset="0x2DF0" />
         <!-- <Blob Name="object_bigpo_Blob_003200" Size="0x350" Offset="0x3200" /> -->
         <!-- <Blob Name="object_bigpo_Blob_003790" Size="0x370" Offset="0x3790" /> -->
-        <DList Name="gBigpoDrawLanternFallingDL" Offset="0x41A0" /> <!-- Original name is "pog_kantera_hahen_model" ("kantera" = "lantern", "hahen" = "fragment") -->
```

Comment:
Ah man got my hopes up the blobs would be figured out...

Reply:
Took a look at this because of this comment. The blobs screamed Vtx, but I couldn't quite figure them out. Then I opened up OoT's `object_po_field` in a hex editor and found it immediately:
![image](https://github.com/zeldaret/mm/assets/12245827/addfe0e0-2a89-4f3f-bd50-bd592f5ffd18)

If you have OoT cloned, you can find them extracted in `object_po_fieldVtx_0039F0.vtx.inc`. If you take a close look at the OoT vertices, you'll see that MM's `object_bigpoVtx_002FF0` and `object_bigpoVtx_003550` are in OoT's `object_po_fieldVtx_0039F0`. So it looks like the blobs are just unused vertices that ZAPD failed to auto-extract, but they were probably all just one Vtx array in the original code.

Reply:
If you want to extract it like OoT, you can do this:
```xml
<Array Name="gIDontKnowWhatToCallThis" Count="283" Offset="0x2FF0">
    <Vtx/>
</Array>
```

But I don't really know what to call this Vtx array.

Reply:
I just went with a simple `gBigPoeUnusedVtx`, let me know if it's worth commenting on the connection to OoT's `object_po_fieldVtx_0039F0`

Reply:
The thing is, there are parts of it that *are* used. In `master` right now, this one Vtx array is divided up into five sections:
- object_bigpoVtx_002FF0 (used)
- object_bigpo_unaccounted_003200 (unused)
- object_bigpoVtx_003550 (used)
- object_bigpo_unaccounted_003790 (unused)
- object_bigpoVtx_003B00 (used)

It's this mixing of used and unused vertices that creates the two weird blobs. So "unused" is not the correct term for this; most of them are indeed used.

Reply:
Actually looked into OoT's code (probably should have done that earlier lol) and I think I understand it now. The way both Poes and Big Poes in Hyrule Field work is that they use this skeleton:
![image](https://github.com/zeldaret/mm/assets/12245827/f337a13a-2798-4b87-ae1f-569f70fd6154)

Now, as you'll notice, this doesn't look like a Big Poe. If they want to render a Big Poe, they replace a few limbs like so:
```c
if (this->actor.params == EN_PO_FIELD_BIG) {
    if (limbIndex == 1) {
        *dList = gBigPoeFaceDL;
    } else if (limbIndex == 8) {
        *dList = gBigPoeCloakDL;
    } else if (limbIndex == 9) {
        *dList = gBigPoeBodyDL;
    }
}
```

So the Vtx array we've been talking about is the regular Field Poe Vtx. That's why most of it is used, but some of it is unused; the limbs that are getting replaced in OoT are just unused entirely in MM.

As for the name of this, I honestly think naming it after Field Poe is the best approach here, preferably with a shortened version of the explanation above. But I'll leave it up to your discretion.

Reply:
Yeah, you're absolutely right. Went ahead and split the vtx, named them with your help, and added a comment explaining the context